### ISSUE TITLE
Possible to convert into vectorized operation

### ISSUE THREAD

**Issue by Sinitca-Aleksandr** [alexandr-1k/StatTools#3]
https://github.com/alexandr-1k/StatTools/blob/7c0106ba874ce079394717039ecb728bfcbf4aa1/StatTools/generators/base_filter.py#L108

**Comment by alexandr-1k** [comment]
Да, спасибо, после замены этого цикла получилось значительно ускориться. С помощью timeit за 1000 прогонов получил разницу более, чем в 10 раз.
